src/views/Dashboard/Customer/Home/MyTest/StakingServiceTab.vue

```diff
@@ -197,7 +197,7 @@ export default {
         "OPEN": "#F60689",
         "CLAIMED": "#5640A5",
         "PROCESSED": "#4CBB65",
-        "WAITINGFORUNSTAKED": "#FAAD15",
+        "WAITINGFORUNSTAKE": "#FAAD15",
```

Comment:
please recheck, i think the previous one (WAITINGFORUNSTAKED) is already correct

Reply:
<img width="868" alt="Screen Shot 2022-02-01 at 13 03 15" src="https://user-images.githubusercontent.com/87308638/151920310-f23140e8-3e8a-4efe-8774-021945900518.png">
based on figma, for the wording it looks like "waiting for unstake"